### ISSUE TITLE
Integrate Playwright

### ISSUE THREAD

**Issue by kylemh** [OperationCode/front-end#1722]
I want to use Playwright for [Automated Visual Testing with Chromatic](https://www.chromatic.com/blog/introducing-visual-testing-for-playwright/) and eventually ditch snapshot tests + ditch Cypress

**Comment by michaelDragomir** [comment]
@kylemh  still want to do this?

**Comment by michaelDragomir** [comment]
There's a PR for this integration. should be done with it soon..currently testing it out...

**Comment by kylemh** [comment]
Yes, I want to do this. I also want to try and use https://github.com/kwonoj/swc-plugin-coverage-instrument so that we can finally use SWC (and ditch Istanbul + Babel). This would allow our builds to be nearly twice as fast and our dev server would be MUCH faster.

**Comment by michaelDragomir** [comment]
@kylemh i'll spend some time on this...and review that repo.

**Comment by kylemh** [comment]
Im willing to ditch coverage too. So focus on Playwright replacing Cypress first. Coverage can come in a later PR. I'm updating some code in the other open PR too. Trying to get us to Next 14 asap

**Comment by michaelDragomir** [comment]
alrighty...i'll open up a new PR and work on replacing Cypress with Playwright.
